### PR TITLE
Feature message box against feature-user-delete

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: src/assets/thirdPartyServices/index.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ export default class SampleServices {
     if (typeof component === "undefined") {
       component = this.serviceOrgIDToComponent[this.generateUniqueID(orgId, serviceId, defaultChainId)];
       if (typeof component === "undefined") {
-        component = <ErrorMsgBox errorMsg="No Component matched" />;
+        component = <MessageBox type="error" errorMsg="No Component matched" />;
```
Comment (chetankothari) [inline src/assets/thirdPartyServices/index.js]: 👍

File: src/components/Login/ForgotPasswordSubmit/index.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ const ForgotPasswordSubmit = ({ history, error, username, forgotPasswordSubmit }
           </button>
         </div>
         <p>fghjk{error}</p>
-        <ErrorMsgBox error={error} />
+        <MessageBox error={error} />
```
Comment (chetankothari) [inline src/components/Login/ForgotPasswordSubmit/index.js]: Looks like you missed `type` here?

File: src/components/UserProfile/UserProfileSettings/index.js [hunk 3]
```diff
@@ -83,9 +83,7 @@ class UserProfileSettings extends Component {
                 sent to your email.
               </p>
             </div>
-
-            <ErrorMsgBox errorMsg={error} />
-
+            <MessageBox errorMsg={error} />
```
Comment (chetankothari) [inline src/components/UserProfile/UserProfileSettings/index.js]: Do we need to pass in `type`?

File: src/components/common/MessageBox/index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import { withStyles } from "@material-ui/styles";
+
+import { useStyles } from "./styles";
+
+const backgroundColor = {
+  error: "error",
+  success: "success",
+};
+
+const MessageBox = ({ classes, errorMsg, type }) => {
```
Comment (chetankothari) [inline src/components/common/MessageBox/index.js]: This could also be named as `Alert`.

File: src/components/common/MessageBox/index.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  if (errorMsg) {
+    return <p className={clsx(classes.messageBox, classes[backgroundColor[type]])}>{errorMsg}</p>;
+  }
+  return null;
+};
+
+MessageBox.propTypes = {
+  type: PropTypes.oneOf(["error", "success"]),
+  showErr: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
+  errorMsg: PropTypes.string,
```
Comment (chetankothari) [inline src/components/common/MessageBox/index.js]: I would go with just `type` and `message` and based on the type I would pick appropriate `styling`. `message` should be a required prop.

**Review by chetankothari — APPROVED** [review]
Great refactoring move. Minor improvements which would further make it more generic.

File: src/components/Login/ForgotPasswordSubmit/index.js (hunk 2 above)
Comment (Vivek205) [reply src/components/Login/ForgotPasswordSubmit/index.js]: type has a default value in Message.defaultProps

File: src/components/UserProfile/UserProfileSettings/index.js (hunk 3 above)
Comment (Vivek205) [reply src/components/UserProfile/UserProfileSettings/index.js]: type has a default value in Message.defaultProps

File: src/components/common/MessageBox/index.js (hunk 5 above)
Comment (Vivek205) [reply src/components/common/MessageBox/index.js]: [done](https://github.com/Vivek205/refactored-dapp/pull/22/commits/39c50b77dc05ab784e7199ff68395eb7a16ceba4)

File: src/components/common/MessageBox/index.js (hunk 4 above)
Comment (Vivek205) [reply src/components/common/MessageBox/index.js]: [done](https://github.com/Vivek205/refactored-dapp/pull/22/commits/ec99abdc8b56f0ccb4e224c4b635cf8633ac6716)

**Comment by Vivek205** [comment]
Changes made based on the sugggestions.. closing the PR

**Pull request closed without merging**